injected/src/features/message-bridge/create-page-world-bridge.js

```diff
@@ -0,0 +1,134 @@
+import {
+    InstallProxy,
+    ProxyNotification,
+    ProxyRequest,
+    ProxyResponse,
+    SubscriptionRequest,
+    SubscriptionResponse,
+    SubscriptionUnsubscribe,
+} from './schema.js';
+
+/**
+ * todo: use the already-captured ones
+ */
+const captured = {
+    dispatchEvent: globalThis.dispatchEvent.bind(globalThis),
+    addEventListener: globalThis.addEventListener.bind(globalThis),
+    removeEventListener: globalThis.removeEventListener.bind(globalThis),
+    CustomEvent: globalThis.CustomEvent,
+    Promise: globalThis.Promise,
+    String: globalThis.String,
+    mathRandom: globalThis.Math.random.bind(globalThis.Math),
+};
+
+/**
+ * This part can be called from any script.
+ * @param {string} featureName
+ * @param {string} token
+ * @return {Pick<import("@duckduckgo/messaging").Messaging, 'notify' | 'subscribe' | 'request'>}
+ */
+export function createPageWorldBridge(featureName, token) {
+    console.log('will createPageWorldBridge for', featureName, 'with token', token);
```

Comment:
Again add this to a debug method, this method could be hooked as it is and steal the token. I do think there's value in obfuscating the token a little and obvious ways to capture it should be avoided.